modules/nixos/monitoring/alert-rules.nix

```
@@ -89,6 +89,11 @@ lib.mapAttrsToList
       description = "{{$labels.host}} failed to (re)start service {{$labels.name}}";
     };
 
+    matrix_alertmanager_receiver_not_running = {
+      condition = ''systemd_units_active_code{name="matrix-alertmanager-receiver.service", instance="localhost:9273", sub!="running"}'';
```

I think you can safely assume that a matrix-alertmanager-receiver should run always no matter on which host:

```suggestion
      condition = ''systemd_units_active_code{name="matrix-alertmanager-receiver.service", sub!="running"}'';
```